Tools/CMake/AMReX_SetupCUDA.cmake

```diff
@@ -86,6 +151,20 @@ foreach (_item IN LISTS _nvcc_arch_flags)
 
 endforeach ()
 
+if (CUDA_LTO)
+    # we replace
+    #   -gencode=arch=compute_NN,code=sm_NN
+    # with
+    #   -gencode=arch=compute_NN,code=lto_NN
```

Comment:
I am not 100% sure about this yet, since I do get warnings of the form
```
nvlink warning : SM Arch ('lto_70') not found in '...'
```
for WarpX object files. But it links and creates a 30MByte smaller binary.

Reply:
Cannot runtime test since drivers on Summit are not up-to-date for CUDA 11.